kitsune/sumo/static/sumo/scss/layout/_document.scss

```diff
@@ -305,3 +305,89 @@
     }
   }
 }
+
+
+.sumo-kb-article-header {
```

Comment:
I would revert this to the way it was before and adjust only for the KB. The `sumo-article-header` is the sumo version of the Protocol Article and not specific to the KB.

Reply:
Ok.